plugins/providers/frontier/client.go

```diff
@@ -232,155 +238,170 @@ func (c *client) GetOrganizations() ([]*Organization, error) {
 	return organizations, err
 }
 
-func (c *client) GrantTeamAccess(resource *Team, userId string, role string) error {
-	body := make(map[string][]string)
-	body["userIds"] = append(body["userIds"], userId)
+func (c *client) GrantGroupAccess(resource *Group, userId string, role string) error {
+	body := make(map[string]string)
+	body["principal"] = fmt.Sprintf("%s:%s", "app/user", userId)
+	body["resource"] = fmt.Sprintf("%s:%s", "app/group", resource.ID)
+	body["roleId"] = role
 
-	endPoint := path.Join(groupsEndpoint, "/", resource.ID, "/", role)
-	req, err := c.newRequest(http.MethodPost, endPoint, body, "")
+	req, err := c.newRequest(http.MethodPost, createPolicyEndpoint, body, "")
 	if err != nil {
 		return err
 	}
 
-	var users []*User
 	var response interface{}
 	if _, err := c.do(req, &response); err != nil {
+		c.logger.Error("Failed to grant access to the user", "Users", userId, req.URL)
```

Comment:
Isn't the log format is `msg, key, val, key, val`? In this `req.URL` is set as `key`, it should have been
`c.logger.Error("Failed to grant access to the user", "Users", userId, "url", req.URL)`? Not sure, please confirm.